package.json

```diff
@@ -25,7 +25,7 @@
       "0.0.0": {
         "cordova-android": ">=4.0.0"
```

Comment:
It's not a showstopper for me. And this might be ignored off course if you are sure.
But should the 0.0.0 version also be locked?

Reply:
I have a bit of a knowledge gap on how exactly this entire Cordova-designed engine restriction works, but I believe that specific information is historical data only.

Only the `"1.3.5"` information is used.

E.g. If a user installed version `1.3.5` that information in the package.json says what is the Cordova Android engine restriction..

If a user installed version 1.3.4 or older, it will never know about the updated restrictions because the `package.json` that is bundled with the old versions will never be updated.

So IMO, I think we do not need to update `0.0.0` because it wouldn't even be used.

Reply:
I think so too, we shouldn't bother about the 0.0.0 version. So feel free to merge.